src/service.ts

```diff
@@ -0,0 +1,80 @@
+const urlRu: string = `https://travel-application-backend.herokuapp.com/countries/ru`;
+const urlEn: string = `https://travel-application-backend.herokuapp.com/countries/en`;
+const urlFr: string = `https://travel-application-backend.herokuapp.com/countries/fr`;
+
+const urlLogin: string = `https://intense-shelf-09539.herokuapp.com/login`;
+const urlRegistration: string = `https://intense-shelf-09539.herokuapp.com/register`;
+
+const urlAddRating: string = `https://intense-shelf-09539.herokuapp.com/addScore`;
+const urlGetRating: string = `https://intense-shelf-09539.herokuapp.com/getScore`;
+
+export const getData = async (lang: string): Promise<Array<Object>> => {
+  let url;
+
+  if (lang === "ru") {
+    url = urlRu;
+  }
+
+  if (lang === "en") {
+    url = urlEn;
+  }
+
+  if (lang === "fr") {
```

Comment:
Several times I saw this construction. This is a good case to put a big object with all the translated texts. However, somebody else already wrote a library for that purpose. https://react.i18next.com/getting-started